app/src/actions/tasks.ts

```diff
@@ -31,15 +33,24 @@ export function loadTasks() {
 export function addTask(task: Task, onComplete: Function) {
   return (dispatch) => {
     delete task._id;
-    return Tasks.add(task)
-      .subscribe((res) => {
-        if (res.length === 1) {
-          dispatch({
-            type: TASK_ADDED,
-            payload: res[0]
-          });
-        }
-      }, null, onComplete); }
+
+    const taskAdd$ = Tasks.add(task).publish();
+    taskAdd$.connect();
```

Comment:
the issue I was having - was once the task was added, wanted to .subscribe in the component to redirect to the correct page. Without using .publish and/or .share - the second .subscribe would trigger another request. 

Talked with Della over lunch, and have an idea of how to handle this now